src/app/screens/Accounts/index.tsx

```diff
@@ -97,6 +130,50 @@ function AccountsScreen() {
             })}
           </tbody>
         </table>
+
+        <Modal
+          closeTimeoutMS={200}
+          isOpen={modalIsOpen}
+          onRequestClose={closeModal}
+          contentLabel="Edit account name"
+          overlayClassName="bg-black bg-opacity-25 fixed inset-0 flex justify-center items-center p-5"
+          className="rounded-lg bg-white w-full max-w-lg"
+        >
+          <div className="p-5 flex justify-between dark:bg-gray-800">
+            <h2 className="text-2xl font-bold dark:text-white">Edit account</h2>
+            <button onClick={closeModal}>
+              <CrossIcon className="w-6 h-6 dark:text-white" />
+            </button>
+          </div>
+
+          <form
+            onSubmit={(e: FormEvent) => {
+              e.preventDefault();
+              updateAccountName(currentAccountId);
```

Comment:
as we already pass in the `currentAccountId` we could also pass in the `newAccountName`? then this method uses consistently data from the arguments.

Reply:
Good point, done here: #705